apps/algorithm/recommend/src/sql_models.py

```diff
@@ -66,5 +66,6 @@ class User_Clicks(SQLModel, table=True):
     click_id: Optional[int] = Field(default=None, primary_key=True)
     user_id: int = Field(foreign_key="users.user_id")
     listing_id: int = Field(foreign_key="listings.listing_id")
+    click_term: str
```

Comment:
What is click_term

Reply:
It's the title of the listing that was clicked on. 

Since there are only 1,500 trained recommendations of the total 50,000 listings, we cannot map the listing_id of a clicked listing to an item in the recommender, since it is likely that item does not exist in the recommender's trained listings. 

So, we instead find similar items based on the title of the listing that was clicked on.